### PR TITLE
Vaulted installments middleware and component

### PR DESCRIPTION
**Pull request by chetanjk**
Adding component for installments

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
         "sync-browser-mocks": "^2.0.8"
     },
     "dependencies": {
-        "@paypal/common-components": "^1.0.10",
+        "@paypal/common-components": "github:chetanjk/paypal-common-components#vaulted-installments",
```
Comment (bluepnume) [inline package.json:81]: Can you revert this line

File: src/zoid/buttons/container.jsx [hunk 2]
```diff
@@ -141,6 +143,8 @@ export function containerTemplate({ uid, props, tag, context, frame, prerenderFr
             <node el={ prerenderFrame } />
 
             <div id={ ID.SMART_MENU } class={ CLASS.SMART_MENU } />
+
+            <div id={ ID.INSTALLMENTS_MODAL } class={ CLASS.INSTALLMENTS_MODAL } />
```
Comment (bluepnume) [inline src/zoid/buttons/container.jsx:147]: I think we can just render this modal up to `body` right? Without introducing a new container

**Comment by bluepnume** [comment]
Looks good, few minor changes + will merge.

File: src/zoid/buttons/container.jsx (hunk 2 above)
Comment (chetanjk) [reply src/zoid/buttons/container.jsx:147]: This actually helps in accessibility where keyboard tabbing moves immediately to installments popup after buttons. We can have a check before rendering if this container exists or else append to body

File: package.json (hunk 1 above)
Comment (chetanjk) [reply package.json]: done

**Comment by bluepnume** [comment]
Can you rebase against master to pick up test fix? Thanks!

**Comment by chetanjk** [comment]
Rebased

**Pull request merged**